.github/workflows/redmine-upkeep.yml

```diff
@@ -19,7 +19,7 @@ on:
         type: boolean
   schedule:
     - cron: '*/30 * * * *'
-  pull_request:
+  pull_request_target:
```

Comment:
Is this for accessing any specific secret? I think GITHUB_TOKEN access permissions can be tweaked. Or is it to prevent that a modified upkeep script could be used to leak secrets?